metabase/metabase
# Fix audit problems

related to https://github.com/metabase/metabase/pull/31680

### Description

It's a prerequisite for node 18 upgrade.
As were suggested at [SO](https://stackoverflow.com/a/73027407) we needed to start with yarn audit fixes otherwise we needed to deal with legacy ssl providers (which were abandoned node17).

### How to verify

- run app locally
- build app locally
- ci should pass
- verify carefully changes in lockfile
